### ISSUE TITLE
[1.12.2] [Request] Pregen command clashing with ChunkPregenerator

### ISSUE THREAD

**Issue by Speiger** [ForgeEssentials/ForgeEssentials#2607]
**Initial Steps**
- [x] I have updated to the latest build from [Jenkins](https://ci.forgeessentials.com/job/FE-1.12.2/) before reporting a bug
- [x] I have checked, if the bug was not already reported by using the [search](https://github.com/ForgeEssentials/ForgeEssentials/issues?q=).
- [x] Attached is my ForgeEssentials/modlist.txt, my latest.log, and my debug.log
- [x] Also, attached are the client logs (if applicable)

**Describe the bug**
ForgeEssentials commands basically clash with Chunkpregen commands. It would be nice if you could use a different prefix to start your pregen command. How about fePregen or have /fe Pregen then other commands...
Would be nice if that could be addressed.

**To Reproduce**
Install ForgeEssentials
Install ChunkPregen
try to use ChunkPregen commands.

**Expected behavior**
If you install both mods that chunkpregen can use its /pregen commands

**Screenshots**

**Desktop (please complete the following information):**

**Additional context**

**Comment by spacebuilder2020** [comment]
That is already a function in 1.12.2.  To remove the use of 'pregen', just go to your `./ForgeEssetials/Commands.cfg` and search for fepregen.

ex:
```
    fepregen {
        S:aliases <
            pregen
            filler
         >
    }
```

**Comment by Speiger** [comment]
Would be nice if you could make that the "default" behavior since someone was just throwing together a pack and ran into the issue where he couldn't use my mod and didn't know why and I had to explain it.

or do you expect this knowledge for everyone to know just magically?

**Comment by spacebuilder2020** [comment]
yea, I can look into it.  I don't want someone who doesn't have your mod to get confused either of course but I will take a look.

**Comment by Speiger** [comment]
Well one solution you could do is move everything into "/fe subcommands" instead of so broadly taking over the base command name space. Its not like you are the only mod that registers commands.

**Comment by Speiger** [comment]
Anyhow since this is a low priority and I don't personally want to deal with this mods conflicts I am going to do the simple solution of warning the player when forge essentials are detected.
This should solve all problems on my side and make it in the worst case your issue. (Which Is reasonable)

Just so you see what is going to be uploaded in the next 5 mins.
![image](https://user-images.githubusercontent.com/2556269/172051807-4858ce72-a688-481c-b3c5-1cc2ae92de1d.png)

And before you asked. I have been burned enough by devs slowing progress down that I have to stress myself out enough just to keep up with their bullshit. (Optifine Dev is the best example)
So this is a preventative measure on my side.

And to reduce your worry. This is shown once.

**Comment by spacebuilder2020** [comment]
yea, that is fair.  They shouldn't be installing it on the client anyway!   Forge Essentials is server side only.

**Comment by spacebuilder2020** [comment]
Keep in mind the reason it is low priority is because Forge Essentials is no longer under active development.

**Comment by Speiger** [comment]
I am fully aware. one of the reasons the Gui/Logger exists with warnings.
For further or existing mod developments that you work on that allow for commands i would kindly suggest, please reduce your footprint in the command namespace. It helps everyone not having clashes.

**Comment by spacebuilder2020** [comment]
Of course.   Also, keep in mind that the pregen command came into existence many years ago (Jun 2, 2015 
), and at the time there were not many other mods with similar commands. https://github.com/ForgeEssentials/ForgeEssentials/commit/0fe8f1fd36453fff494395548dd70019b0b49b50

**Comment by spacebuilder2020** [comment]
And honestly, I really should just put a GUI blocker of my own in cause people go and install FE on the client and then complain that things are not working.

**Comment by spacebuilder2020** [comment]
Also, the solution I was thinking about was to simply detect that your mod was installed and disable the pregen alias.

**Comment by Speiger** [comment]
Then you would also have to detect Chunky and a few other pregenerator mods that exist out there.
That these use the namespace is kinda reasonable since they do the exact same as my mod.
(And with same i mean that's their only purpose)

And yeah the command is older. That's one thing I can give.

**Comment by spacebuilder2020** [comment]
But yes, it started to be an issue a few years back with FTBU so restructured the commands to all start with fe and have their old names as aliases.  I am a bit reluctant to remove the old command alias because it has been part of the mod for so long but it isn't hard to do.

**Comment by spacebuilder2020** [comment]
I will also mention that the other reason I am reluctant to make arbitrary changes is I did do so for the maker of the Corail grave mod (as he forced his mod to crash when FE was installed even when we politely informed him that all users had to do was disable the afterlife module.)  #2506 Sometime after I made the change, his mod got removed from curse for TOS violations. https://www.reddit.com/r/feedthebeast/comments/hdo8v7/corail_tombstone_malicious_behavior/
